docs/_static/cookie_notice.js

```diff
@@ -0,0 +1,29 @@
+const send_ga = () => {
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+   })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-124167090-6', 'auto');
+  ga('send', 'pageview');
+}
+
+let cookieAccepted = localStorage.cookieAccepted;
```

Comment:
Food for thought: consider tracking the exact version of the privacy policy accepted -- and not extrapolate the consent to the "current policy" to "all future policies".